assets/js/base/context/cart-checkout/checkout/index.js

```diff
@@ -45,8 +45,8 @@ export const CheckoutProvider = ( {
 					activePaymentMethod={ initialActivePaymentMethod }
 				>
 					<ShippingDataProvider>
-						{ children }
 						<CheckoutProcessor />
+						{ children }
```

Comment:
It's odd how you had to move this (I saw the commit message). I would have expected the reverse to be needed...

Reply:
Yeah, that's because I moved the `onCheckoutCompleteError` hook to the CheckoutProcessor instead of the CheckoutBlock, so when calling `scrollToTop`, it needed to be on top.

But thinking about it twice, I think that effect belongs more to the block than the processor, so I moved it back in 56164430a05ea5ca5587bd5e2462d453496d43c1.